Packet++/header/TcpReassembly.h

```diff
@@ -208,6 +224,34 @@ class TcpStreamData
 };
 
 
+/**
+ * @struct TcpReassemblyConfiguration
+ * A structure for configuring the TcpReassembly class
+ */
+struct TcpReassemblyConfiguration
+{
+	/** The flag indicating whether to remove the connection data after a connection is closed */
+	bool removeConnInfo;
+
+	/** How long the closed connections will not be cleaned up. The value is expressed in seconds. If the value is set to 0 then TcpReassembly should use the default value */
+	uint32_t closedConnectionDelay;
+
+	/** The maximum number of items to be cleaned up per one call of purgeClosedConnections. If the value is set to 0 then TcpReassembly should use the default value */
+	uint32_t maxNumToClean;

```

Comment:
It'd be useful to mention that this parameter is only relevant if `removeConnInfo` is equal to `true`